ckanext/ontario_theme/templates/internal/gtm_pageview.html

```diff
@@ -13,13 +13,14 @@
     }
 
     // Pushed when a dataset page or a resource page is viewed
-    if ('{{this_pkg}}'.length > 0) {
+    if ('{{this_pkg_access_level}}'.length > 0 ||
+        '{{this_pkg_title}}'.length > 0) {
```

Comment:
why are you checking length of pkg title?

Reply:
previous check was `this_pkg`, which contained both `access_level` and `title`. This was an attempt to recreate the same check. I see the point that either being > 0 will mean that both are valid.